releasetool/github.py

```diff
@@ -22,7 +22,15 @@
 
 import jwt
 import requests
-from cryptography.hazmat.backends import default_backend
+
+import cryptography
+
+CRYPTOGRAPHY_VERSION = cryptography.__version__
+
+if int(CRYPTOGRAPHY_VERSION[0:2]) < 42:
+    from cryptography.hazmat.backends import default_backend
+else:
+    from cryptography.hazmat.primitives import serialization
```

Comment:
Do we really want to try to maintain support for multiple versions of cryptography?

Reply:
Fixed in 91c2e4055e5a2b1746d8bd18f7f1a792a20724f9 and 69ddaef56519dd461e5ee4ecde5c7ac9c99ad5e4